ansible/roles/satellite_master_supervisor/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+---
+# Copyright 2015 TWO SIGMA OPEN SOURCE, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: install pip
+  apt: name=python-pip update_cache=true state=latest
+  become: true
+  become_method: sudo
+  tags:
+    - supervisor
+    - preconditions
+    - master
+
+- name: install supervisor
+  pip: name=supervisor
+  become: true
+  become_method: sudo
+  tags:
+    - supervisor
+    - preconditions
+    - master
+
+- name: place supervisor config
+  template: src=supervisord.conf.j2 dest={{ satellite_install_dir }}/supervisord.conf
+  tags:
+    - supervisor
+    - master
+
+- name: create supervisor task directory
+  file: path={{ satellite_install_dir }}/supervisor.d state=directory
```

Comment:
Could you make the supervisor directory a separate directory from the satellite directory? This will make it easier to use these scripts for existing supervisord-based deployments.